tests/util/localregistry/cmd/localregistry.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"istio.io/istio/mixer/cmd/shared"
```

Comment:
Why do we have a dependency on mixer?

Reply:
To use shared.Printf, shared.Fatalf... I think that library should be moved out of mixer

Reply:
If we go with the approach of returning errors as I suggest elsewhere, then I think it would obviate the need for these functions.